src/pytest_env/plugin.py

```diff
@@ -2,14 +2,28 @@
 from __future__ import annotations
 
 import os
+import sys
 
 import pytest
 
+if sys.version_info >= (3, 8):  # pragma: no cover (py38+)
+    from typing import Final
+else:  # pragma: no cover (<py38)
+    from typing_extensions import Final
+
+_DEFAULT_FLAG: Final[str] = "D"
+_RAW_FLAG: Final[str] = "R"
+
+# Only used for testing purposes
+_TEST1_FLAG: Final[str] = "__PYTEST1__"
+_TEST2_FLAG: Final[str] = "__PYTEST2__"
```

Comment:
It's impossible to tell from output whether both R and D flags have been applied as their functionality is mutually exclusive. 

However, for future maintainability (in case more flags are added, or changed), we must make sure that the flag "platform" works correctly. A proper way of doing this probably would be to write a separate module for parsing flags, but I shied away, because parsing is integrated with ini, so it's better to test parsing end-to-end, rather than a subpart of the whole pipeline.

Reply:
Why do we need to care if both have been applied? If they set different values, you can conclude no? I'll have a think but I don't like the current solution.

Reply:
__PYTEST__ removed, as per previous discussion

Reply:
> Why do we need to care if both have been applied?

For future maintainability, it's about creating a platform for parsing flags, where if future library maintainers were to create an additional flag (say, `S:` to not trim whitespace for env var values), then this additional flag would play nice with other flags, rather than introduce silent bugs

Reply:
Already said that I plan to freeze the pytest.ini format. All future extension points will be made via new interface powered by pyproject.toml that doesn't require custom parsing. Come to think of it maybe we should freeze it now, and add the raw support via that.